torch/onnx/_internal/fx/passes/type_promotion.py

```diff
@@ -30,7 +30,7 @@
 from torch.fx.node import Node  # noqa: F401
 
 from torch.onnx._internal import _beartype
-from torch.onnx._internal.fx import _pass, diagnostics
+from torch.onnx._internal.fx import _pass, diagnostics, type_utils
```

Comment:
To make it more clear

```suggestion
from torch.onnx._internal.fx import _pass, diagnostics, type_utils as fx_type_utils
```